app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @conferences_available = Conference.with_roles([:admin, :organizer, :cfp], current_user)
```

Comment:
@differentreality, me and @nTraum discussed Friday that, starting from `@comments = Comment.accessible_by(current_ability)`,  how can we sort comments by event_title, and after that call @comments in the view, in such manner that they will be displayed like Github does. So that was about the above question.
In the meantime, I think I found an workaround solution, and please tell me if this is the right one.
Starting from accessible comments, I take events ids, get all events with those ids, sorted after title:
````events_ids = Comment.accessible_by(current_ability).pluck(:comment able_id)
@events = Event.where(id: events_ids).order(title: :desc)
````
In this manner, it seems much easier to iterate through events and show them in the view as:
````@events.each do |event|
  = event.title
  - event.comment_threads.each do |comment|
    = comment.body
```